server/lib/flow-event.js

```diff
@@ -23,7 +24,98 @@ var HOSTNAME = os.hostname();
 var MAX_DATA_LENGTH = 100;
 var VERSION = 1;
 
-module.exports = function (event, data, request) {
+const FLOW_BEGIN_EVENT_TYPES = /^flow\.[a-z_-]+\.begin$/;
+const FLOW_ID_KEY = config.get('flow_id_key');
+const FLOW_ID_EXPIRY = config.get('flow_id_expiry');
+
+const ENTRYPOINT_PATTERN = /^[\w\.-]+$/;
+const SERVICE_PATTERN = /^(sync|content-server|none|[0-9a-f]{16})$/;
+const VALID_FLOW_EVENT_PROPERTIES = [
```

Comment:
I guess so, do you want me to change it?

I opted for an array because it makes iteration cleaner and I find the explicit naming of `key` and `pattern` more readable.